src/pages/preview/[...slug].tsx

```diff
@@ -27,6 +28,10 @@ const ContentPage: NextPage<ContentPageProps> = ({
     relatedContent,
     previewMode,
 }) => {
+    if (!contentItem) {
+        return <Error statusCode={404} />;
+    }
+
```

Comment:
This is kinda unrelated, but Patrick mentioned that it might be better for us to only have preview routes accessible on staging to prevent them from technically being public. Do you think we should just also return a 404 here if we are on prod?

Reply:
Yeah, fair; I did find it risky to expose preview even though people need the correct URL to access;

I can update this, but do we need to check with Greg?

Reply:
Yeah i think it would be best to loop him in

Reply:
You mention the authors can potentially experience breaking changes on the staging app, but I would like to know if that is something we are encountering now or have in the past. Or are we seeing potential in that happening

Reply:
It seems like the consensus is to keep the preview exclusive to the staging app. I will update it accordingly!

Reply:
> You mention the authors can potentially experience breaking changes on the staging app, but I would like to know if that is something we are encountering now or have in the past.

Not really. I just wrote that to be precautions, as the staging app (i.e., main branch) should largely be stable anyway.